code/modules/mob/living/carbon/carbon.dm

```diff
@@ -357,18 +357,31 @@
 	if(HAS_TRAIT(H, TRAIT_SKELETONIZED) && (!H.w_uniform) && (!H.wear_suit))
 		H.play_xylophone()
 
+/mob/living/carbon/can_be_flashed(intensity = 1, override_blindness_check = 0)
+
+	var/obj/item/organ/internal/eyes/E = get_int_organ(/obj/item/organ/internal/eyes)
+	. = ..()
+
+	if((E && E.status & ORGAN_DEAD) || !.)
```

Comment:
```suggestion
	if((E && (E.status & ORGAN_DEAD)) || !.)
```
Not sure if it's needed, prob not, but this will at least make it clear it'll work so you don't have to think about it while reading it